### PR TITLE
Ventilators

### PR DESCRIPTION
**Pull request by galitLukin**
I apologize in advance for:
1. no comments at all
2. the allocation file is super long. I've split it up as much as I can tonight without getting errors. When I'm more awake, I'll split it up more.
I suggest we reiterate on this once there is more time when I fix the two issues above. I'm sorry this will be hard to read and understand.

It doesn't look perfect but I think it's good enough for release. Mainly, please ensure it works as expected.

Only necessary thing missing is documentation. If we're doing it through a pdf, please make a separate pr for it or send me it and I will add it on the documentation page which now says bla (on purpose so that we dont deploy it empty)

### CODE REVIEW

**Comment by adelarue** [comment]
My first comment after taking a look is: this looks AMAZING! The most impressive thing is that all the pieces are there, and I think the code is modular enough that we can get to the final product just by tweaking things around.

**Comment by adelarue** [comment]
When I inspect element, I see a few errors. Most of them seem useless (e.g. use marginBottom instead of margin-bottom). Only one may need fixing:

Error: Each child in a list should have a unique "key" prop. See https://fb.me/react-warning-keys for more information.

**Comment by leobix** [comment]
Wonderful!!!! Congratulations!!! Looks fantastic!

A few comments:
1. Indeed we need to fill the documentation page. So far I propose to write on the documentation page, a very simple: You can download the full documentation [here](pdf link). Arthur I think could make the PR for the pdf.

2. The dropdown for Surge Supply Availability is not aligned because of Availability. Subject to be discussed then: should we remove Availability?

3. The coloured map corresponding to shortage does not allow to really distinguish between excess supply or not with the current colorbar. We already discussed that for the consistency it is better to keep same colors, but maybe here it makes sense to switch to a different style like Green/Red?

4. When I want to display the to and From transfers, no state choice appears for me. Maybe this because the python index.py did not work properly in this regard? I would just like someone to double check on this.

**Comment by adelarue** [comment]
Responding to @leobix:
1. Done
2. Done
3. I second this, but I was not part of the conversation today. This can be fixed in v2.
4. You are seeing nothing because there are no transfers on those days. Try changing the date to 4/7 or 4/8

Additionally @galitLukin I fixed an issue with the table of transfers sometimes bleeding into the next row (downloads) by setting the height to auto.

**Comment by leobix** [comment]
Thank you Arthur!!
To follow up on 4.
If possible we should therefore include something like "no transfers happening" because user would need to see it is not a bug? I think it is good to go for tomorrow morning but propose to discuss more on this when we have time.

**Comment by adelarue** [comment]
> Thank you Arthur!!
> To follow up on 4.
> If possible we should therefore include something like "no transfers happening" because user would need to see it is not a bug? I think it is good to go for tomorrow morning but propose to discuss more on this when we have time.

Yes, this is a high-priority item for v2.

**Comment by leobix** [comment]
I hope I am doing nothing wrong, but it appears I merge things. Please tell me if I am messing up anything!

**Comment by leobix** [comment]
Another comment: when choosing the date I cannot beyond April 20, is it normal?

File: ventilators/utils.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 model_visual = {
                 "Shortage":"Shortage",
                 "Supply": "Supply",
-                "Demand":"Projected Ventilator Demand"
```
Comment (galitLukin) [inline ventilators/utils.py]: If you will change this, be consistent with the UI on the top of the page please

Comment (leobix) [reply ventilators/utils.py]: Been changed to previous version :) Was a mistake on my side

**Comment by galitLukin** [comment]
Thanks to all of you for looking at this so late. Holly and I will show this to Dimitris tomorrow morning and update. Good night!

**Comment by galitLukin** [comment]
New stuff after call with Dimitris:
1. Split into more files for readability
2. Many text changes
3. Change to one dropdown
4. Change second timeline
5. Make table static with option to filter

**Comment by galitLukin** [comment]
Approved via slack by Arthur, Agni, and Holly.

**Pull request merged**